aws-opentelemetry-distro/src/amazon/opentelemetry/distro/otlp_udp_exporter.py

```diff
@@ -0,0 +1,109 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import socket
+from logging import Logger, getLogger
+from typing import Dict, Optional, Sequence, Tuple
+
+from typing_extensions import override
+
+from opentelemetry.exporter.otlp.proto.common.metrics_encoder import encode_metrics
+from opentelemetry.exporter.otlp.proto.common.trace_encoder import encode_spans
+from opentelemetry.sdk.metrics._internal.aggregation import AggregationTemporality
+from opentelemetry.sdk.metrics._internal.export import MetricExportResult
+from opentelemetry.sdk.metrics._internal.point import MetricsData
+from opentelemetry.sdk.metrics.export import MetricExporter
+from opentelemetry.sdk.metrics.view import Aggregation
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
+
+DEFAULT_ENDPOINT = "127.0.0.1:2000"
+PROTOCOL_HEADER = '{"format":"json","version":1}\n'
+PROTOCOL_DELIMITER = "\n"
+
+_logger: Logger = getLogger(__name__)
+
+
+class UdpExporter:
```

Comment:
It is using UDP protocol but we customize the exporter for Lambda FluxPump, so the names of these exporters can be `LambdaFluxpumpExporter`

Reply:
Discussed offline that we don't explicitly need to call it lambda exporter. Simply UdpExporter is acceptable.